src/status_im2/contexts/onboarding/identifiers/profile_card/style.cljs

```diff
@@ -0,0 +1,33 @@
+(ns status-im2.contexts.onboarding.identifiers.profile-card.style
+  (:require [react-native.reanimated :as reanimated]
+            [quo2.components.avatars.user-avatar.style :as avatar-style]))
+
+(defn card-container
+  [background-style]
+  {:style [(reanimated/apply-animations-to-style
+            {}
+            {:padding-horizontal 12
+             :padding-top        12
+             :padding-bottom     12
+             :flex               1
+             :border-radius      16})
+           background-style]})
+
+(defn opacity
+  [opacity normal-style]
+  {:style (reanimated/apply-animations-to-style
+           {:opacity opacity}
+           normal-style)})
```

Comment:
Normal style is just empty, please remove arg and hard code {}

Reply:
Done